cime_config/machines/config_machines.xml

```diff
@@ -1217,6 +1223,102 @@
       <env name="NETCDF_PATH">/nfs/gce/projects/climate/software/netcdf/4.8.0c-4.3.1cxx-4.5.3f-parallel/mpich-3.4.2/gcc-11.1.0</env>
       <env name="PNETCDF_PATH">/nfs/gce/projects/climate/software/pnetcdf/1.12.2/mpich-3.4.2/gcc-11.1.0</env>
     </environment_variables>
+    <environment_variables mpilib="openmpi">
+      <!-- We currently don't have modules for HDF5, NetCDF & PnetCDF -->
+      <env name="LD_LIBRARY_PATH">/nfs/gce/projects/climate/software/openmpi/4.1.3/gcc-11.1.0/lib:$ENV{LD_LIBRARY_PATH}</env>
+      <env name="PATH">/nfs/gce/projects/climate/software/openmpi/4.1.3/gcc-11.1.0/bin:$ENV{PATH}</env>
+      <env name="ZLIB_PATH">/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.5.0/zlib-1.2.11-smoyzzo</env>
+      <env name="HDF5_PATH">/nfs/gce/projects/climate/software/hdf5/1.12.1/openmpi-4.1.3/gcc-11.1.0</env>
+      <env name="NETCDF_PATH">/nfs/gce/projects/climate/software/netcdf/4.8.0c-4.3.1cxx-4.5.3f-parallel/openmpi-4.1.3/gcc-11.1.0</env>
+      <env name="PNETCDF_PATH">/nfs/gce/projects/climate/software/pnetcdf/1.12.2/openmpi-4.1.3/gcc-11.1.0</env>
+    </environment_variables>
+    <environment_variables SMP_PRESENT="TRUE">
+      <env name="OMP_STACKSIZE">64M</env>
+    </environment_variables>
+    <environment_variables>
+      <env name="PERL5LIB">/nfs/gce/projects/climate/software/perl5/lib/perl5</env>
+    </environment_variables>
+  </machine>
+
+  <machine MACH="anlgce">
+    <DESC>ANL CELS General Computing Environment (Linux) workstation</DESC>
+    <NODENAME_REGEX>compute-(240|386)-[0-9][0-9]</NODENAME_REGEX>
```

Comment:
Does this mean that the user needs to explicitly specify the machine name (create_newcase etc) for ubuntu 18 machines?

Reply:
No. On compute-01 and compute-02 the machine name will be identified as anlgce-ub18 and on other GCE nodes the machine name will be identified as anlgce.

Reply:
This is because anlgce-ub18 is defined before anlgce in the machine file and "compute-386-01|compute-386-02" will be matched first.

Reply:
ok, thanks. Verified that it works with compute-01 without specifying the machine name when creating the case.